test/Npgsql.Tests/MultipleHostsTests.cs

```diff
@@ -1,1185 +1,1185 @@
-            Assert.That(connections[6].Port, Is.EqualTo(pR2.Port));
-            Assert.That(connections[7].Port, Is.EqualTo(s2.Port));
-            Assert.That(connections[8].Port, Is.EqualTo(pR1.Port));
-            Assert.That(connections[9].Port, Is.EqualTo(s1.Port));
-            Assert.That(connections[10].Port, Is.EqualTo(pR2.Port));
-            Assert.That(connections[11].Port, Is.EqualTo(s2.Port));
-            break;
-        case TargetSessionAttributes.Primary:
-        case TargetSessionAttributes.PreferPrimary:
-            Assert.That(connections[0].Port, Is.EqualTo(pRw1.Port));
-            Assert.That(connections[1].Port, Is.EqualTo(pR1.Port));
-            Assert.That(connections[2].Port, Is.EqualTo(pRw2.Port));
-            Assert.That(connections[3].Port, Is.EqualTo(pR2.Port));
-            Assert.That(connections[4].Port, Is.EqualTo(pRw1.Port));
-            Assert.That(connections[5].Port, Is.EqualTo(pR1.Port));
-            Assert.That(connections[6].Port, Is.EqualTo(pRw2.Port));
-            Assert.That(connections[7].Port, Is.EqualTo(pR2.Port));
-            Assert.That(connections[8].Port, Is.EqualTo(pRw1.Port));
-            Assert.That(connections[9].Port, Is.EqualTo(pR1.Port));
-            Assert.That(connections[10].Port, Is.EqualTo(pRw2.Port));
-            Assert.That(connections[11].Port, Is.EqualTo(pR2.Port));
-            break;
-        case TargetSessionAttributes.Standby:
-        case TargetSessionAttributes.PreferStandby:
-            Assert.That(connections[0].Port, Is.EqualTo(s1.Port));
-            Assert.That(connections[1].Port, Is.EqualTo(s2.Port));
-            Assert.That(connections[2].Port, Is.EqualTo(s1.Port));
-            Assert.That(connections[3].Port, Is.EqualTo(s2.Port));
-            Assert.That(connections[4].Port, Is.EqualTo(s1.Port));
-            Assert.That(connections[5].Port, Is.EqualTo(s2.Port));
-            Assert.That(connections[6].Port, Is.EqualTo(s1.Port));
-            Assert.That(connections[7].Port, Is.EqualTo(s2.Port));
-            Assert.That(connections[8].Port, Is.EqualTo(s1.Port));
-            Assert.That(connections[9].Port, Is.EqualTo(s2.Port));
-            Assert.That(connections[10].Port, Is.EqualTo(s1.Port));
-            Assert.That(connections[11].Port, Is.EqualTo(s2.Port));
-            break;
-        }
-    }
-
-    static string MultipleHosts(params PgPostmasterMock[] postmasters)
-        => string.Join(",", postmasters.Select(p => $"{p.Host}:{p.Port}"));
-
-    class DisposableWrapper : IAsyncDisposable
-    {
-        readonly IEnumerable<IAsyncDisposable> _disposables;
-
-        public DisposableWrapper(IEnumerable<IAsyncDisposable> disposables) => _disposables = disposables;
-
-        public async ValueTask DisposeAsync()
-        {
-            foreach (var disposable in _disposables)
-                await disposable.DisposeAsync();
-        }
-    }
-}
+﻿// using YBNpgsql.Internal;
```

Comment:
It was commented because it is an upstream test and takes the value of `Load Balance Hosts` as boolean value. Since I made the parameter as enum, the build started failing. Uncommented the test, and modified all the test cases with targeted changes for load balance values.